libs/libxx/libxx_cxa_atexit.cxx

```diff
@@ -120,6 +123,7 @@ extern "C"
           return on_exit(__cxa_callback, alloc);
         }
       else
+
```

Comment:
don't need change

Reply:
Yes I will do a proper clean up for this big PR tomorrow. There are many changes that can be merged into one etc.